netci.groovy

```diff
@@ -321,6 +333,16 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
     // Pull request builds.  Generally these fall into two categories: default triggers and on-demand triggers
     // We generally only have a distinct set of default triggers but a bunch of on-demand ones.
     def osGroup = getOSGroup(os)
+
+    // Get crossgen out of the way
+    if (doCrossgen) {
+        assert (os == 'Windows_NT' || os in Constants.crosslist)
+        // Add trigger phrase for Checked x64 pri1 Crossgen builds
+        if (configuration == 'Checked' && architecture == 'x64' && scenario == 'pri1') {
```

Comment:
Checked and Release, got it. No problem.

For Pri1, it would be invoked via phrase - "test {os} pri1 r2r please.", where os can be Ubuntu, OSX, Windows_NT, or CentOS7.1